app/connectors/CustomsDeclareExportsMovementsConnector.scala

```diff
@@ -31,47 +32,42 @@ import scala.concurrent.{ExecutionContext, Future}
 import scala.util.{Failure, Success}
 
 @Singleton
-class CustomsDeclareExportsMovementsConnector @Inject()(appConfig: AppConfig, httpClient: HttpClient) {
+class CustomsDeclareExportsMovementsConnector @Inject()(appConfig: AppConfig, httpClient: HttpClient)(
+  implicit ec: ExecutionContext
+) {
 
   private val logger = Logger(this.getClass)
 
   private val CustomsDeclareExportsMovementsUrl = s"${appConfig.customsDeclareExportsMovements}"
 
   private val movementSubmissionUrl: PartialFunction[ActionType, String] = {
-    case Arrival            => s"$CustomsDeclareExportsMovementsUrl${appConfig.movementArrivalSubmissionUri}"
-    case Departure          => s"$CustomsDeclareExportsMovementsUrl${appConfig.movementDepartureSubmissionUri}"
-    case DucrAssociation    => s"$CustomsDeclareExportsMovementsUrl${appConfig.movementConsolidationAssociateUri}"
-    case DucrDisassociation => s"$CustomsDeclareExportsMovementsUrl${appConfig.movementConsolidationDisassociateUri}"
-    case ShutMucr           => s"$CustomsDeclareExportsMovementsUrl${appConfig.movementConsolidationShutMucrUri}"
+    case Arrival   => s"$CustomsDeclareExportsMovementsUrl${appConfig.movementArrivalSubmissionUri}"
+    case Departure => s"$CustomsDeclareExportsMovementsUrl${appConfig.movementDepartureSubmissionUri}"
+    case DucrAssociation | DucrDisassociation | ShutMucr =>
+      s"$CustomsDeclareExportsMovementsUrl${appConfig.movementConsolidationUri}"
   }
 
   private val CommonMovementsHeaders =
     Seq(HeaderNames.CONTENT_TYPE -> ContentTypes.XML(Codec.utf_8), HeaderNames.ACCEPT -> ContentTypes.XML(Codec.utf_8))
 
-  def sendArrivalDeclaration(
-    requestXml: String
-  )(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[HttpResponse] = postRequest(Arrival, requestXml)
+  private val JsonHeaders = Seq(HeaderNames.CONTENT_TYPE -> ContentTypes.JSON, HeaderNames.ACCEPT -> ContentTypes.JSON)
 
-  def sendDepartureDeclaration(
-    requestXml: String
-  )(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[HttpResponse] = postRequest(Departure, requestXml)
+  def sendArrivalDeclaration(requestXml: String)(implicit hc: HeaderCarrier): Future[HttpResponse] =
+    postRequest(Arrival, requestXml)
 
-  def sendAssociationRequest(
-    requestXml: String
-  )(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[HttpResponse] = postRequest(DucrAssociation, requestXml)
+  def sendDepartureDeclaration(requestXml: String)(implicit hc: HeaderCarrier): Future[HttpResponse] =
```

Comment:
parameter name ?

Reply:
departure and arrival is still based on the xml